srcpkgs/obsidian/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'obsidian'
+pkgname=obsidian
+version=0.12.3
+revision=1
+makedepends="asar"
+depends="zlib hicolor-icon-theme fuse electron12"
+short_desc="Obsidian is a powerful knowledge base that works on top of a local folder of plain text Markdown files"
+maintainer="Soc Virnyl S. Estela <renegan.ronin@gmail.com>"
+license="custom:Commercial"
```

Comment:
> Customer shall not [...] distribute or share the Services or Software or make any of them available for access by third parties

~ https://obsidian.md/terms

needs restricted=, unlikely to be accepted at all

Reply:
Added `restricted=yes`. Thanks for pointing that out.